.github/workflows/benchmark-cli.yml

```diff
@@ -7,29 +7,30 @@ jobs:
   benchmark:
     name: Benchmark CLI
     runs-on: ubuntu-latest
+    if: startsWith(github.event.action, 'pull_request') || startsWith(github.event.action, 'issue_comment')
```

Comment:
This adds the issue_comment as a valid event action so the cli can trigger benchmarks and external PRs.